runtime/src/lib.rs

```diff
@@ -337,7 +356,8 @@ construct_runtime!(
 		Sudo: pallet_sudo::{Pallet, Call, Config<T>, Storage, Event<T>},
 		Creditcoin: pallet_creditcoin::{Pallet, Call, Storage, Config<T>, Event<T>},
 		Difficulty: pallet_difficulty::{Pallet, Call, Config<T>, Storage},
-		Rewards: pallet_rewards::{Pallet, Storage, Event<T>}
+		Rewards: pallet_rewards::{Pallet, Storage, Event<T>},
+		Scheduler: pallet_scheduler,
```

Comment:
Needs a test to safeguard against possible changes in behavior of this pallet:

1) Schedule an action in the future
2) roll_by() that many blocks
3) assert action was executed

Reply:
WIP